StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -245,6 +272,27 @@ extension PaymentSheet {
 
                 return availabilityStatus == .supported
             }
+
+            if let paymentMethodOrder = configuration.paymentMethodOrder?.map({ $0.lowercased() }) {
```

Comment:
Does this need to be done in the client? Does the endpoint not take the `paymentMethodOrder` param and return them already ordered?

Reply:
It doesn't, although I agree it should be done on the endpoint!  I'm hoping to get that into the server implementation of this feature for GA.